wp-content/themes/core/components/page/Controller.php

```diff
@@ -0,0 +1,78 @@
+<?php
+declare( strict_types=1 );
+
+namespace Tribe\Project\Templates\Components\page;
+
+use Tribe\Project\Templates\Factory_Method;
+use Tribe\Project\Templates\Models\Breadcrumb;
+
+class Controller {
+	use Factory_Method;
```

Comment:
What is the purpose of having Factory_Method is all of these Controllers. I'm missing something.